### PR TITLE
Docs: Update UG formatting + Errors

### PR DESCRIPTION
**Pull request by RyanNgWH**
Mainly updated formatting & phrasing issues. Also addressed some errors in the UG

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -252,43 +262,108 @@ You must either specify an exercise name, or use one or more of the default supp
...
+
+  If `burpees` already exists, please refer to this [warning](#fitadd-overwrite-warning) for the behaviour of this command.
+
+* `fitadd 1 n/burpees s/3 r/5 b/30` - Adds `burpees` exercise to the client at index `1` with the following values:
+  * `Sets`: 3 (specified value).
+  * `Reps`: 5 (specified value).
+  * `Break between sets`: 30 seconds (specified value).
+
+  If `burpees` already exists, please refer to this [warning](#fitadd-overwrite-warning) for the behaviour of this command.
+
```
Comment (dillontkh) [inline docs/UserGuide.md:332]: The overwrite warning for each example seems a little repetitive.

File: docs/UserGuide.md [hunk 2]
```diff
@@ -348,13 +423,13 @@ Format: `find [n/NAME] [p/PHONE] [e/EMAIL] [a/ADDRESS] [w/RANGE] [h/RANGE] [nt/N
...
-    * e.g. `find roys n/roy` will use n/roy to search for a client's name instead of roys.
+    * e.g. `find roys n/roy` will use `n/roy` to search for a client's name instead of `roys`.
 </div>
 
 
 Examples:
-* `find n/Wendy` returns `Wendy Son` and `Wendy Kim`
-* `find n/Wendy t/Lover` returns `Wendy` (`Name` contains `Wendy` and is tagged with `Lover`)
+* `find n/Wendy` - Finds all clients whose name contains `Wendy`
+* `find n/Wendy t/Lover` - Finds all clients who name contains `Wendy` and is tagged with `Lover`
```
Comment (dillontkh) [inline docs/UserGuide.md:432]:
```suggestion
* `find n/Wendy` - Finds all clients whose names contain `Wendy`
* `find n/Wendy t/Lover` - Finds all clients whose names contain `Wendy` and are tagged with `Lover`
```

**Review by dillontkh — CHANGES_REQUESTED** [review]
LGTM! Just some minor grammatical errors

**Review by ruijietay — APPROVED** [review]
LGTM

File: docs/UserGuide.md (hunk 1 above)
Comment (ruijietay) [reply docs/UserGuide.md:332]: I agree, would it be better to put it all under one sentence instead?

Comment (RyanNgWH) [reply docs/UserGuide.md:332]: Alright, made the necessary changes already. Please review it again :)

**Review by dillontkh — APPROVED** [review]
LGTM!

**Review by jjchee77 — APPROVED** [review]
LGTM!

**Pull request merged**